cmd/img_mgmt/port/zephyr/src/zephyr_img_mgmt.c

```diff
@@ -73,7 +73,8 @@ zephyr_img_mgmt_flash_check_empty(uint8_t fa_id, bool *out_empty)
         rc = flash_area_read(fa, addr, data, bytes_to_read);
         if (rc != 0) {
             flash_area_close(fa);
-            return MGMT_ERR_EUNKNOWN;
+            LOG_ERR("Read Error, Might be Corrupted");
+            return MGMT_ERR_ECORRUPT;
```

Comment:
Is there a retcode for ECC error that could be detected?

Reply:
There is not :/